conda-forge.yml

```diff
@@ -1,6 +1,4 @@
 build_platform:
-  linux_aarch64: linux_64
-  linux_ppc64le: linux_64
```

Comment:
Sadly this doesn't seem to work on this feedstock. Please see PR ( https://github.com/conda-forge/implicit-feedstock/pull/85 ) for initial work in this direction